packages/aws-cdk/lib/cdk-toolkit.ts

```diff
@@ -394,14 +394,15 @@ export class CdkToolkit {
 
   private async selectStacksForDiff(stackNames: string[], exclusively?: boolean) {
     const assembly = await this.assembly();
-    const stacks = await assembly.selectStacks(stackNames, {
+
+    const idsToValidate = process.env.STACKS_TO_VALIDATE ? process.env.STACKS_TO_VALIDATE.split(';') : stackNames;
```

Comment:
Did we confirm that the method of using an environment variable is going to scale to large sizes? Or are we going to use wildcards?

Reply:
I've tried with values as long as 50K characters and it worked fine.